code/__defines/subsystems.dm

```diff
@@ -54,17 +54,18 @@
 #define INIT_ORDER_ATOMS 11
 #define INIT_ORDER_LANGUAGE 10
 #define INIT_ORDER_MACHINES 9
-#define INIT_ORDER_SHUTTLE 3
 #define INIT_ORDER_TIMER 1
 #define INIT_ORDER_DEFAULT 0
 #define INIT_ORDER_AIR -1
-#define INIT_ORDER_MINIMAP -2
-#define INIT_ORDER_ASSETS -3
+#define INIT_ORDER_ALARM -2
+#define INIT_ORDER_MINIMAP -3
+#define INIT_ORDER_ASSETS -4
 #define INIT_ORDER_ICON_SMOOTHING -5
 #define INIT_ORDER_OVERLAY -6
 #define INIT_ORDER_XKEYSCORE -10
 #define INIT_ORDER_STICKY_BAN -10
 #define INIT_ORDER_LIGHTING -20
+#define INIT_ORDER_SHUTTLE -21
 #define INIT_ORDER_SQUEAK -40
```

Comment:
squeak? Whats this? looks like it was there before

Reply:
It was, it looks like someone pulled defines, but not subsystems, so there may be more unused defines i guess, i haven't looked in that stuff in this PR, since this PR is quite big already, so, maybe in future PRs. Anyway, https://github.com/tgstation/tgstation/blob/master/code/controllers/subsystem/squeak.dm that's what it could be.